scripts/deploy.ts

```diff
@@ -1,15 +1,100 @@
+import {ethers as ethers2, Wallet} from 'ethers';
 import {ethers} from 'hardhat';
+import {ARBITRUM_NETWORK} from '../deploy/constants';
+import {L1LPTGateway__factory} from '../typechain';
 
-async function main() {
-  const Greeter = await ethers.getContractFactory('Greeter');
-  const greeter = await Greeter.deploy('Hello, Hardhat!');
+export function getArbitrumCoreContracts(l2: ethers2.providers.BaseProvider) {
+  return {
+    arbRetryableTx: new ethers.Contract(
+        ARBITRUM_NETWORK.rinkeby.arbRetryableTx,
+        require('../test/utils/abis/ArbRetryableTx.json').abi,
```

Comment:
If we have these interfaces check in could we also do the same thing here with the interface instead of the ABI?

Reply:
Yeah better to have interface in contract than abi in json